<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="TEXT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>871</width>
     <height>621</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SHOW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAPTURE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NEXT">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Zomm-</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1922</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ngang2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel_3">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>40</y>
     <width>271</width>
     <height>701</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NEXT_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn thiết bị</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NEXT_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Tùy chọn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NEXT_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm mới</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAPTURE_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>VideoRecoder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAPTURE_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NEXT_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>IN PHIẾU KHÁM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NEXT_7">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Đóng</string>
   </property>
  </widget>
  <zorder>TEXT</zorder>
  <zorder>imgLabel</zorder>
  <zorder>SHOW</zorder>
  <zorder>NEXT</zorder>
  <zorder>imgLabel_3</zorder>
  <zorder>NEXT_2</zorder>
  <zorder>NEXT_4</zorder>
  <zorder>NEXT_5</zorder>
  <zorder>imgLabel_2</zorder>
  <zorder>CAPTURE</zorder>
  <zorder>CAPTURE_2</zorder>
  <zorder>CAPTURE_3</zorder>
  <zorder>NEXT_3</zorder>
  <zorder>NEXT_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
